terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/resources/org/terasoluna/gfw/common/codelist/i18n/simpleI18nCodeList.xml

```diff
@@ -119,10 +120,40 @@
         </entry>
       </util:map>
     </property>
+    <property name="fallbackTo" value="en" />
   </bean>
 
+  <bean id="CL_testFallbackTo" class="org.terasoluna.gfw.common.codelist.i18n.SimpleI18nCodeList">
+    <property name="rows">
+      <util:map>
+        <entry key="en">
+          <util:map>
+            <entry key="0" value="Sun." />
+            <entry key="1" value="Mon." />
+            <entry key="2" value="Tue." />
+            <entry key="3" value="Wed." />
+            <entry key="4" value="Thu." />
+            <entry key="5" value="Fri." />
+            <entry key="6" value="Sat." />
+          </util:map>
+        </entry>
+        <entry key="ja">
+          <util:map>
+            <entry key="0" value="日" />
+            <entry key="1" value="月" />
+            <entry key="2" value="火" />
+            <entry key="3" value="水" />
+            <entry key="4" value="木" />
+            <entry key="5" value="金" />
+            <entry key="6" value="土" />
+          </util:map>
+        </entry>
+      </util:map>
+    </property>
+    <property name="fallbackTo" value="en" />
```

Comment:
Since an error occurs in the bean definition, only adding a test pattern is done.
We are also changing method names to make test patterns easier to understand.

6f6e2c0b2a30914dc9724ed7716853d087c8be1b